priv/templates/phx.gen.auth/context_functions.ex

```diff
@@ -284,16 +280,16 @@
   ## Token helper
 
   defp update_<%= schema.singular %>_and_delete_all_tokens(changeset) do
-    %{data: %<%= inspect schema.alias %>{} = <%= schema.singular %>} = changeset
-
-    with {:ok, %{<%= schema.singular %>: <%= schema.singular %>, tokens_to_expire: expired_tokens}} <-
-           Ecto.Multi.new()
-           |> Ecto.Multi.update(:<%= schema.singular %>, changeset)
-           |> Ecto.Multi.all(:tokens_to_expire, <%= inspect schema.alias %>Token.by_<%= schema.singular %>_and_contexts_query(<%= schema.singular %>, :all))
-           |> Ecto.Multi.delete_all(:tokens, fn %{tokens_to_expire: tokens_to_expire} ->
-             <%= inspect schema.alias %>Token.delete_all_query(tokens_to_expire)
-           end)
-           |> Repo.transaction() do
-      {:ok, <%= schema.singular %>, expired_tokens}
-    end
+    Repo.transact(fn ->
+      with {:ok, <%= schema.singular %>} <- Repo.update(changeset) do
+        query = <%= inspect schema.alias %>Token.by_<%= schema.singular %>_and_contexts_query(<%= schema.singular %>, :all)
+        tokens_to_expire = Repo.all(query)
+
+        tokens_to_expire
+        |> <%= inspect schema.alias %>Token.delete_all_query()
+        |> Repo.delete_all()
+
+        {:ok, {<%= schema.singular %>, tokens_to_expire}}
```

Comment:
I changed it to pass that tuple through all places where we previously had the three-tuple.